app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+  def create
+    #is this the proper way to pass in params?
+
+    new_review_data = review_params
```

Comment:
You're calling `review_params` correctly, but you don't actually need to set it to a variable! Instead, you can just call `review_params` directly on line 8: `review = @vendor.reviews.new(review_params)`